bundles/org.openhab.core.model.core/pom.xml

```diff
@@ -14,6 +14,12 @@
 
   <name>openHAB Core :: Bundles :: Model Core</name>
 
+  <properties>
+    <maven.compiler.source>${oh.model.java.version}</maven.compiler.source>
+    <maven.compiler.target>${oh.model.java.version}</maven.compiler.target>
+    <maven.compiler.compilerVersion>${oh.model.java.version}</maven.compiler.compilerVersion>
+  </properties>
```

Comment:
Did you also try to add these properties to the [model-specific-profile](https://github.com/openhab/openhab-core/blob/2ae3137fec410a7056cdffbe285b9967811b5095/bundles/pom.xml#L187-L272) to avoid duplicating them?

Reply:
Nice. I didn't know that profile existed.

Reply:
That seemed to have worked out well! :+1: